beets/art.py

```diff
@@ -69,7 +69,10 @@ def embed_item(log, item, imagepath, maxwidth=None, itempath=None,
     except IOError as exc:
         log.warning(u'could not read image file: {0}', exc)
         return
-    item.try_write(path=itempath, tags={'images': [image]})
+
+    write = config['import']['write'].get(bool)
+    if write:
```

Comment:
Heh, sorry. Copied the code from elsewhere in the repo. My python atrophies quickly.